pkg/minikube/localpath/localpath_test.go

```diff
@@ -61,3 +66,95 @@ func TestHasWindowsDriveLetter(t *testing.T) {
 		}
 	}
 }
+
+func TestMiniPath(t *testing.T) {
+	var testCases = []struct {
+		env, basePath string
+	}{
+		{"/tmp/.minikube", "/tmp/"},
+		{"/tmp/", "/tmp"},
+		{"", homedir.HomeDir()},
+	}
+	originalEnv := os.Getenv(MinikubeHome)
+	for _, tc := range testCases {
+		defer func() { // revert to pre-test env var
```

Comment:
The defer func needs to be before the for loop

Reply:
fixed